.gitignore

```diff
@@ -6,4 +6,5 @@ coverage
 .env
 junit.xml
 frontend/build
+openapi3.yaml
```

Comment:
What is this file? Should I have to generate it somehow?

Reply:
I can remove that file from the gitignore. It used to be generated when we were using a different library to serve/generate the API documentation.